### PR TITLE
Update permissions.yml reverse flash daniel west

### PR DESCRIPTION
**Pull request by Leedler268**

### CODE REVIEW

**Comment by mythguy1226** [comment]
you changed the prefix for reverseflash eobard thawne change it back
also get rid of the extra space at the top of the file

**Comment by Leedler268** [comment]
See hero.ms file for proof of powers.

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -8758,6 +8758,52 @@ groups:
         - ch.alias.haste2
         - ch.alias.haste3
         - ch.alias.haste4
+ reverseflashdanielwest:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: Not enough spaces here

**Comment by Leedler268** [comment]
PLEASE READ THIS COMMENT:
IF GDE DECIDES TO ADD BACK THE POWERS HE REMOVED, PLEASE REMEMBER TO ADD THEM HERE.

**Pull request merged**